python/torch_mlir/tools/opt/__main__.py

```diff
@@ -0,0 +1,30 @@
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# Also available under a BSD-style license. See LICENSE.
+
+import os
```

Comment:
Please add a comment like the one at https://github.com/llvm/torch-mlir/blob/09af3b6030d8d0c0ee8a80840734224d5c4b82a3/python/torch_mlir/tools/import_onnx/__main__.py#L6-L15

Reply:
Done.